platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateFileManager.java

```diff
@@ -216,6 +216,115 @@ public int getTaskQueueSize() {
         return taskQueue.size();
     }
 
+    /**
+     * Method to be called when a state has been successfully written to disk.
+     *
+     * @param reservedState the state that was written to disk
+     * @param directory     the directory where the state was written
+     * @param start         the nano start time of the state writing process
+     */
+    private void stateWrittenToDisk(
```

Comment:
Need to add `reservedState.stateSavedToDisk()` in this method

Reply:
@poulok Done

I also moved things around a bit, after discussing with @cody-littley That `stateWrittenToDisk` shouldn't be getting called for states written out-of-band

Reply:
hmmm, interesting. In that case, the method should probably be renamed. It's not intuitive to have a method called `stateWrittenToDisk` that is only called for some states that are written to disk.

Reply:
I renamed it to `stateWrittenToDiskInBand`, and tweaked some javadocs to try to provide some additional clarity.